### PR TITLE
[VPDQ] Add python test github workflow and clean up test

### PR DESCRIPTION
**Pull request by tigerchensh**
Summary
---------

Add github work flow for vpdq python library, since now it can be easily locally installed through vpdq-release.py script. Adjust test parameters with CPP github workflow.

Test Plan
---------

github workflow

### CODE REVIEW

File: vpdq/python/tests/test_vpdq_hash.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 SAMPLE_HASH_FOLDER = Path("vpdq/sample-hashes")
 SAMPLE_VIDEOS = Path("tmk/sample-videos")
 DISTANCE_TOLERANCE = 10
-QUALITY_TOLERANCE = 50
```
Comment (BarrettOlson) [inline vpdq/python/tests/test_vpdq_hash.py:12]: blocking: Why are we updating this to 80 again?

File: vpdq/python/tests/test_util.py [hunk 2]
```diff
@@ -1,8 +1,9 @@
 # Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
 import vpdq
+import typing as t
 
 
-def read_file_to_hash(input_hash_filename: str) -> list[vpdq.VpdqFeature]:
+def read_file_to_hash(input_hash_filename: str) -> t.List[vpdq.VpdqFeature]:
```
Comment (BarrettOlson) [inline vpdq/python/tests/test_util.py:6]: nit: t.Sequence is a bit more flexible

**Review by BarrettOlson — CHANGES_REQUESTED** [review]
Thanks for adding this!
Can you explain your reasoning for updating QUALITY_TOLERANCE?

File: vpdq/python/tests/test_vpdq_hash.py (hunk 1 above)
Comment (tigerchensh) [reply vpdq/python/tests/test_vpdq_hash.py:12]: "Adjust test parameters with CPP github workflow." CPP reg test uses 80. 50 may fail on various ffmpeg versions caused by that low-quality video.

Comment (BarrettOlson) [reply vpdq/python/tests/test_vpdq_hash.py:12]: Having a test QUALITY_TOLERANCE higher than the used in prod `VPDQ_QUALITY_THRESHOLD = 50` doesn't make sense to me. (Same goes for what we do in C++ land)
I would prefer we skip the low-quality video for a regression test as it is more import the test reflects the production state of the library.

Comment (tigerchensh) [reply vpdq/python/tests/test_vpdq_hash.py:12]: The DISTANCE_TOLERANCE is lower than the used in prod. I can adjust those to be the same as the prod data. Is it better?

Comment (BarrettOlson) [reply vpdq/python/tests/test_vpdq_hash.py:12]: Sure, for the reg test, let's match prod defaults.
Was it a deliberate decision not to do so originally?

Comment (tigerchensh) [reply vpdq/python/tests/test_vpdq_hash.py:12]: Yes, we had discussion on this for smaller DISTANCE_TOLERANCE for validation process.

**Comment by tigerchensh** [comment]
Thanks for the review. Has adjusted test parameters align with prod environment.

File: vpdq/python/tests/test_vpdq_hash.py (hunk 1 above)
Comment (BarrettOlson) [reply vpdq/python/tests/test_vpdq_hash.py:12]: Ah I see, I view implementation validation as different from regression testing:
validation: two implementation of the vpdq algorithm ideally output nearly identical hashes (good to have much for strict params)
regression: adjustments, refactors, etc to code do not result in large changes in production behavior (can be more strict or just match production params)

**Review by BarrettOlson — APPROVED** [review]

**Pull request merged**